config/urls.py

```diff
@@ -25,6 +25,8 @@
     path("", include(("app.main.urls", "main"), namespace="main")),
     path("healthcheck/", include("app.healthcheck.urls")),
     path("admin/", admin.site.urls),
+    path("api-auth/", include("rest_framework.urls")),
```

Comment:
We don't want dedicated login and logout views. Only useful if we used the template browser for the API.